build.gradle

```
@@ -17,9 +17,10 @@ dependencies {
     annotationProcessor group: 'org.springframework.boot', name: 'spring-boot-configuration-processor'
     compile 'org.springframework.boot:spring-boot-configuration-processor'
     compile 'org.springframework.boot:spring-boot-starter-security'
-    compile group: 'org.eclipse.jgit', name: 'org.eclipse.jgit', version: '5.7.0.202003110725-r'
     //compile 'org.springframework.security:spring-security-cas'
     //compile 'org.jasig.cas.client:cas-client-support-springboot:3.6.1'
+    compile group: 'org.springframework.security', name: 'spring-security-oauth2-client', version: '5.3.1.RELEASE'
+    compile group: 'org.eclipse.jgit', name: 'org.eclipse.jgit', version: '5.7.0.202003110725-r'
```

ъуъ неконсистентненько 
```suggestion
    compile 'org.springframework.security:spring-security-oauth2-client:5.3.1.RELEASE'
    compile 'org.eclipse.jgit:org.eclipse.jgit:5.7.0.202003110725-r'
```